### PR TITLE
Fix spelling mistakes found by codespell

### PR DESCRIPTION
**Pull request by andrewimeson**
This splits out #2422 to have just the typo fix commit

All of these typos were identified by [codespell](https://github.com/codespell-project/codespell)

### CODE REVIEW

**Review by aallan — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Before starting writing your contribution to the documentation, you should take
 
 ## Type of Content
 
-We welcome contributions from the community, ranging from correcting small typos all the way through to adding entire new sections to the documentation. However, going forward we're going to be fairly targetted about what sort of content we add to the documentation. We are looking to keep the repository, and the documentation, focussed on Raspberry Pi-specific things, rather than having generic Linux or computing content.
+We welcome contributions from the community, ranging from correcting small typos all the way through to adding entire new sections to the documentation. However, going forward we're going to be fairly targeted about what sort of content we add to the documentation. We are looking to keep the repository, and the documentation, focused on Raspberry Pi-specific things, rather than having generic Linux or computing content.
```
Comment (RuffaloLavoisier) [inline CONTRIBUTING.md:21]: @aallan 
I know it as a British spelling. we're a British company. 😄

Comment (lurch) [reply CONTRIBUTING.md:21]: https://www.future-perfect.co.uk/grammar-tips/just-plain-wrong/is-it-targetted-or-targeted suggests that "targeted" is correct?

Comment (RuffaloLavoisier) [reply CONTRIBUTING.md:21]: https://wikidiff.com/targeted/targetted  Which one is right?

Comment (aallan) [reply CONTRIBUTING.md:21]: Target's accent is on the first syllable, hence **targeting** is correct, as per Oxford English Dictionary and @helenlynn who is the arbiter of all things. There are exceptions to the rule, for instance, if the word ends in an 'l' but this is not one of them.

**Pull request merged**